static/knowledgebase_toc.json

```diff
@@ -906,10 +906,11 @@
     },
     {
```

Comment:
Not relevant to this PR but I think this can be removed as well, it's generated at build time. Will PR separately.